pkgs/data/icons/fluent-icon-theme/default.nix

```
@@ -0,0 +1,73 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, gtk3
+, hicolor-icon-theme
+, jdupes
+, roundedIcons ? false
+, blackPanelIcons ? false
+, colorVariant ? []
```

```suggestion
, colorVariants ? []
```
Shouldn't this be called `colorVariants`, since you can specify more than one?

---

At the same time one could rationalize against this, because it is expected from the user to install 1 of the available color variants and not several at once. I don't know to be honest.

---

I have `grep`ed for `colorVariant` in the `pkgs/data/icons/` directory and it looks like all the ones that have a `colorVariants` argument (`qogir-icon-theme`, `tela-circle-icon-theme`, `vimix-icon-theme` and `colloid-icon-theme`) also have it in plural. So I would write this one in plural too, to fit with the others.

Furthermore: if the user may use only one at a time, then this argument should not be a list but only the one color variant.

---

Done!